smarts/core/traffic_history.py

```diff
@@ -89,6 +94,12 @@ def target_speed(self) -> float:
         query = "SELECT value FROM Spec where key='speed_limit_mps'"
         return self._query_val(float, query)
 
+    @cached_property
+    def ego_vehicle_id(self) -> int:
+        query = "SELECT id FROM Vehicle WHERE is_ego_vehicle = 1"
+        ego_id = self._query_val(int, query)
```

Comment:
We added the `is_ego_vehicle`entry to 0 as default for datasets not having this entry.

Reply:
No, I mean, when there are no vehicles with `is_ego_vehicle = 1`, the `_query_val` call will return `None` (just confirmed by looking at it), which means `ego_id` will be returned to the caller as `None`.   It looks like you're immediately trying to convert it to a `str` in the history vehicles replacement script.  We should probably handle the case of `None` there too (and anywhere else this is called).

Reply:
This query in particular is only used in Waymo dataset which will always record which vehicle was Waymo ego (since it was recorded by it). We can add a flag checking if the value returned is not None, but I guess we will just hijack a random vehicle if by chance it is not mentioned in the dataset.